editor/src/components/menubar/menubar.tsx

```diff
@@ -174,9 +180,17 @@ export const Menubar = betterReactMemo('Menubar', () => {
         </Tooltip>
       </FlexColumn>
       {isFeatureEnabled('Performance Test Triggers') ? (
-        <Tile style={{ marginTop: 12, marginBottom: 12 }}>
-          <a onClick={onTriggerScrollTest}>P S</a>
-        </Tile>
+        <React.Fragment>
+          <Tile style={{ marginTop: 12, marginBottom: 12 }}>
+            <a onClick={onTriggerScrollTest}>P S</a>
+          </Tile>
+          <Tile style={{ marginTop: 12, marginBottom: 12 }}>
+            <a onClick={onTriggerResizeTest}>P R</a>
+          </Tile>
+          <Tile style={{ marginTop: 12, marginBottom: 12 }}>
+            <a onClick={onTriggerSelectionTest}>P E</a>
+          </Tile>
+        </React.Fragment>
```

Comment:
If we're only using these for puppeteer, we could presumably style these to have zero width and height so that they're not visible to an actual user

Reply:
They're not rendered unless the trigger is enabled right?

Reply:
Yeah but it's enabled by default